Ryujinx/Ui/ApplicationLibrary.cs

```diff
@@ -69,30 +76,77 @@ public static void Init(List<string> AppDirs, Keyset keySet, SystemState.TitleLa
                     continue;
                 }
 
-                DirectoryInfo AppDirInfo = new DirectoryInfo(appDir);
-                foreach (FileInfo App in AppDirInfo.GetFiles())
+                string[] apps = Directory.GetFiles(appDir, "*.*", SearchOption.AllDirectories);
+                foreach (string app in apps)
```

Comment:
```suggestion
                foreach (string app in Directory.GetFiles(appDir, "*.*", SearchOption.AllDirectories))
```
and remove `string[] apps` since you don't use it at another place.